docs/source/en/model_doc/electra.md

```diff
@@ -14,66 +14,96 @@ rendered properly in your Markdown viewer.
-  available ELECTRA models, however. This means that the discriminator may be loaded in the
-  [`ElectraForMaskedLM`] model, and the generator may be loaded in the
-  [`ElectraForPreTraining`] model (the classification head will be randomly initialized as it
-  doesn't exist in the generator).
-
-## Resources
-
-- [Text classification task guide](../tasks/sequence_classification)
-- [Token classification task guide](../tasks/token_classification)
-- [Question answering task guide](../tasks/question_answering)
-- [Causal language modeling task guide](../tasks/language_modeling)
-- [Masked language modeling task guide](../tasks/masked_language_modeling)
-- [Multiple choice task guide](../tasks/multiple_choice)
+# ELECTRA
+
+[ELECTRA](https://huggingface.co/papers/2003.10555) is a clever alternative to traditional masked language models like BERT. Instead of just masking tokens and asking the model to predict them, ELECTRA trains two models working together: a generator and a discriminator. The generator replaces some tokens with plausible alternatives, and the discriminator (the model you'll actually use) learns to detect which tokens are original and which were replaced.
+
+This approach is super efficient because ELECTRA learns from every single token in the input, not just the masked ones. That's why even the small ELECTRA models can match or outperform much larger models while using way less computing resources.
+
+You can find all the original ELECTRA checkpoints under the [ELECTRA release](https://huggingface.co/collections/google/electra-release-64ff6e8b18830fabea30a1ab) collection.
+
+> [!TIP]
+> Click on the right sidebar for more examples of how to use ELECTRA for different language tasks like sequence classification, token classification, and question answering.
+
+The example below demonstrates how to use ELECTRA for text classification tasks with [`Pipeline`] or the [`AutoModel`] class.
+
+<hfoptions id="usage">
+<hfoption id="Pipeline">
+
+```py
+from transformers import pipeline
+
+classifier = pipeline("text-classification", model="google/electra-small-discriminator")
+result = classifier("This restaurant has amazing food!")
+print(result)
+```
+
+</hfoption>
+<hfoption id="AutoModel">
+
+```py
+from transformers import AutoTokenizer, AutoModelForSequenceClassification
+import torch
+tokenizer = AutoTokenizer.from_pretrained("google/electra-small-discriminator")
+model = AutoModelForSequenceClassification.from_pretrained("google/electra-small-discriminator")
+inputs = tokenizer("ELECTRA is more efficient than BERT", return_tensors="pt")
+with torch.no_grad():
+outputs = model(**inputs)
+logits = outputs.logits
+print(logits)
+```
+
+</hfoption>
+</hfoptions>
+
+Quantization reduces the memory burden of large models by representing the weights in a lower precision. Refer to the [Quantization](../quantization/overview) overview for more available quantization backends.
```

Comment:
Probably not necessary to include a quantization section here since ELECTRA isn't that large